src/components/ModalExport.tsx

```diff
@@ -205,6 +210,106 @@ const ExportDropdown: FC<ExportDropdownProps> = ({ selected, onSelect }) => {
   )
 }
 
+const ModalExportContainer = tw.section`
+ flex justify-center
+ mb-4
+`
+
+const ActionWrapper = tw.div`
+  flex justify-center
+  mt-14 mb-5
+`
+
+const ClipboardWrapper = tw.div`
+  text-center
+`
```

Comment:
In general, we want to name things based on what they do rather than where they are used. Since this component is simply a `<div>` with centered text, we can call it `TextCenter`. This makes is easier to reuse and easier to understand at the call site without having to look up the definition.

If it makes sense, you may be able to apply this principle to other style definitions.

Let me know if there is a Tailwind or styled component convention that I'm not aware of 🙂.

Reply:
I prefer naming things for what they are here. What they do is already defined by their styling. My preference is based on the fact that on most projects I try to find things based on what they are before I try to understand what they do. For example: If I wanted to refactor or make changes to the clipboard related components in `ModalExport` after some days, it would be easier to get the right components based on what they are. I won't be able to find things easily if we name them based on what they do.  I agree that naming things based on what they do is useful for reuseable styled-components. `ClipboardWrapper` happens to be a `div` with centered text. But we would have to change the name if later we add some other styles to it. Also with `tailwind`  we already have access to great reusable CSS utilities, so we don't have to create separate reusable components like `TextCenter`.

I see this approach commonly used in `styled-components` community too.

Reply:
> My preference is based on the fact that on most projects I try to find things based on what they are before I try to understand what they do. For example: If I wanted to refactor or make changes to the clipboard related components in `ModalExport` after some days, it would be easier to get the right components based on what they are. I

Okay. So the component names serve as a form of documentation that makes it easier to read the component hierarchy and identify components based on their role.

> But we would have to change the name if later we add some other styles to it.

Or extend/mix-in other styles, though it looks like atomic styled components are more idiomatic.

> Also, it won't be easier to always name things based on what they do, especially when styles are complex.

That's a good point. A lot of components are simply an ad hoc set of styles for that specific situation.

Let's give it a try. I was initially cautious because the "semantic" approach to styling in HTML was a disaster in the late 90's and early 2000's. It caused a proliferation of location-specific classes, and it was the standard approach recommended for years. However styled components are co-located, scoped, and get compiled down in the build step so I think it's not a problem.